packages/smooth_app/lib/pages/product/edit_ingredients_page.dart

```diff
@@ -131,12 +129,52 @@ class _EditOcrPageState extends State<EditOcrPage> {
   }
 
   Future<bool> _updateText(final String text) async {
+    final Product minimalistProduct =
+        _helper.getMinimalistProduct(_product, text);
+    final String uniqueId =
```

Comment:
The same for the id generation

Reply:
Since everytime i require a unique id with those parameters so I am creating at places where i need them